### ISSUE TITLE
No keybinding setting on Firefox?

### ISSUE THREAD

**Issue by TRSx80** [stsquad/emacs_chrome#176]
Thanks for sharing this extension!  :beers:  This is just the neatest thing since sliced bread!  You have saved my life from endless opening of File menu when in my mind all I was really trying to do was `forward-word`!  :laughing: 

But I do not see any option to allow a keyboard shortcut in the extension's settings?

My life is already much better even without that, but it would be nice to get working.  Sort of like the cherry on top if you will.  :smile:

**Comment by stsquad** [comment]
The key configuration is handled in one place for all extensions. Go to about:addons, click the cog at the top of the list and select "Manage Extension Shortcuts".

**Comment by TRSx80** [comment]
Thanks for fast reply!

I never knew about that menu before...

Inside, I did of course find the relevant settings.  Both the default (I think it was Alt-e (M-e)) as well as my attempt to change that to S-M-e (or M-E, if you will) ended in the keys being intercepted by Firefox (Edit menu).

However, I was successful by changing the keybinding to C-e.

Sorry for wasting your time to have to teach me about Firefox, mate!  And thanks again for this extension!  :beers:

Closing the issue.
